In NixOS chroot builds, there is no root account.  So configure should
not rely on its existence in deciding whether to use getpwuid().

diff -rc -x '*~' rcs-5.7-orig/src/conf.sh rcs-5.7/src/conf.sh
*** rcs-5.7-orig/src/conf.sh	1995-06-16 08:19:24.000000000 +0200
--- rcs-5.7/src/conf.sh	2010-02-12 15:22:37.000000000 +0100
***************
*** 821,827 ****
  	a= z=
  	cat >a.c <<EOF
  #include "$A_H"
! int main() { exitmain(!getpwuid(0)); }
  EOF
  	$PREPARE_CC || exit
  	if ($CL a.c $L && $aout) >&2
--- 821,827 ----
  	a= z=
  	cat >a.c <<EOF
  #include "$A_H"
! int main() { exitmain(!getpwuid(getuid())); }
  EOF
  	$PREPARE_CC || exit
  	if ($CL a.c $L && $aout) >&2
